### PR TITLE
Add helper to load font

### PR DESCRIPTION
**Pull request by promag**
Originally submitted as https://github.com/bitcoin-core/gui-qml/pull/49.

### CODE REVIEW

File: src/qt/guiutil.cpp [hunk 1]
```diff
@@ -272,6 +272,12 @@ bool hasEntryData(const QAbstractItemView *view, int column, int role)
     return !selection.at(0).data(role).toString().isEmpty();
 }
 
+void loadFont(const QString& file_name)
```
Comment (hebasto) [inline src/qt/guiutil.cpp:275]: Why not return `id` for generality?

From Qt [docs](https://doc.qt.io/qt-5.12/qfontdatabase.html#addApplicationFont):
>  An ID is returned that can be used to remove the font again with `removeApplicationFont()` or to retrieve the list of family names contained in the font.

File: src/qt/guiutil.cpp [hunk 2]
```diff
@@ -272,6 +272,12 @@ bool hasEntryData(const QAbstractItemView *view, int column, int role)
...
+{
+    int id = QFontDatabase::addApplicationFont(file_name);
```
Comment (hebasto) [inline src/qt/guiutil.cpp:277]:
```suggestion
    const int id = QFontDatabase::addApplicationFont(file_name);
```
?

File: src/qt/bitcoin.cpp [hunk 3]
```diff
@@ -492,7 +491,7 @@ int GuiMain(int argc, char* argv[])
 #endif
 
     BitcoinApplication app;
-    QFontDatabase::addApplicationFont(":/fonts/monospace");
+    GUIUtil::loadFont(":/fonts/monospace");
```
Comment (hebasto) [inline src/qt/bitcoin.cpp:494]:
```suggestion
    GUIUtil::loadFont(QStringLiteral(":/fonts/monospace"));
```

**Comment by luke-jr** [comment]
Why do we want to abort if it fails??

**Pull request merged**